misc/scripts/deploy-tools.sh

```diff
@@ -210,6 +211,8 @@ function deployESXInstall {
 }
 
 function deployESXPost {
+    # restart hostd so that esxcli extension file is read
+    $SSH $TARGET $HOSTD restart
```

Comment:
Couldn't come across a better way to do this. This is what the esxcli doc mentions about discovering extensions
```
In ESX 5.0 this step is performed once by the hostagent daemon when it starts up. Any commands available to the system during hostd startup will be registered as valid esxcli commands.
```

Ignore the 5.0. The documentation is old.

I ll check again if we can do anything better than this.